core/domain/fs_domain.py

```diff
@@ -324,13 +378,14 @@ def get(self, filepath):
             raise IOError('File %s not found.' % (filepath))
         return file_stream.read()
 
-    def commit(self, filepath, raw_bytes, mimetype=None):
+    def commit(
+        self, filepath: str, raw_bytes: bytes, mimetype: str = ' ') -> None:
```

Comment:
```suggestion
    def commit(
        self, filepath: str, raw_bytes: bytes, mimetype: str = ' '
    ) -> None:
```
why is mimetype `' '`

Reply:
It is empty by default.

Reply:
Empty string is not a correct value, why cannot you allow `None`?